src/IO/Messenger.php

```diff
@@ -468,7 +468,28 @@ private function decorateTemplateSource(Template\TemplateSource $templateSource)
             return $name;
         }
 
-        return sprintf('%s <fg=gray>(%s)</>', $description, $name);
+        return sprintf(
```

Comment:
This does not apply to packages without a description (see the early return on line 467).

Reply:
Nah, that's obviously a bug. I addressed in 01992b2c6b09e3ca9093c7ed384e8504f5ff857a.